### DISCUSSION TITLE
Improve an Osu!Lazer Experience for Mobile , Tablet , IOS User

### DISCUSSION THREAD

**Discussion by Skibidi1927** [ppy/osu#23724] [Ideas] [answered]
I recommend
STD - adding Side 2 Transparent Buttons Right side or Left side
For Osu!std mode that will made the game easier to play for mobile / Tablet / Ios User and also use a another finger drag and play
MANIA - adding little transparent mobile tiles 1k , 2k , 3k , 4k , 5k , 6k and 7k
If it hard to imagine for 4K make it by place an button 4 button that split the screen in a 1:4 aspect ratio and key mode that need space bar to play i recommend for place it like a keyboard below these button

You can also turn it on or off if want to connect an Keyboard by turn it in setting
In Mobile Category if have in future

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/4201
